client-v2/src/shared/util/externalArticle.ts

```diff
@@ -48,3 +50,22 @@ export function hasMainMediaVideoAtom(
     isVideo(mainBlockElement)
   );
 }
+
+/**
+ * Create a selector to answer the question -- is the article last modified field older than the given date?
+ * This function is liberal in what it accepts -- if either date is invalid/missing, it returns `true`.
+ */
+export const createSelectIsArticleStale = <State>(
+  selectArticleById: (state: State, id: string) => ExternalArticle | undefined
+) => (state: State, id: string, dateStr: string | undefined): boolean => {
```

Comment:
Could we type `dateStr` as a `Date`?

Reply:
We can, but that would be mean every piece of consuming code having to roll their own date creation, along with the invalid case. I'm not sure that's preferable -- what do you think? An alternative might be to detail the format of the date string in a comment -- it's the same format we expect from CAPI.